blocks/library/gallery/style.scss

```diff
@@ -17,6 +17,21 @@
 		}
 	}
 
+	// Cropped
+	&.crop-true .blocks-gallery-image {
+		img {
+			width: 100%;
+			height: 100%;
+			object-fit: cover;
+		}
+
+		// IE10+ hack
```

Comment:
I wasn't able to test this in IE11 yet. Couldn't get docker running. Going to try again with a different approach.

But it should apply to 10 and 11. Edge supports `object-fit`.